tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/amqp/AmqpSendReceiveInterceptorTest.java

```diff
@@ -26,8 +26,12 @@
 import org.apache.activemq.transport.amqp.client.AmqpReceiver;
 import org.apache.activemq.transport.amqp.client.AmqpSender;
 import org.apache.activemq.transport.amqp.client.AmqpSession;
+import org.apache.felix.resolver.util.ArrayMap;
```

Comment:
Can we use an alternative std java map such as hash map, tree map. rather than introducing a new code dependency. (felix is used for the maven bundlers but not currently as a code dep. doing a search of the code base)